src/index.html

```diff
@@ -4,10 +4,49 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product Card</title>
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+         class="card__image"
+         src="images/imac.jpeg"
+         alt="Apple iMac 27 Retina 5K Monoblock"
+      >
+      <h2 class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__reviews-block">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+      <div class="price">
+        <p class="price__text">Price:</p>
+        <p class="price__amount">$2,199</p>
+      </div>
```

Comment:
it should be only 2 BEM blocks for this task (card and stars)
```suggestion
      <div class="card__price">
        <p class="card__price-text">Price:</p>
        <p class="card__price-amount">$2,199</p>
      </div>
```